src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+    </div>
+    <h3>Adaptive Courseware</h3>
+    <div class="resource-row">
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+    </div>
+    <h2>Student Resources</h2>
+    <div class="resource-row">
+        <a class="resource-box">
+            Getting Started Guide
+        </a>
+        <a class="resource-box">
+            Student Solution Manual
+        </a>
+        <a class="resource-box">
+            Free! Veritas Tutors How to Succeed in Physics
+        </a>
+    </div>
+    <div class="cta">
+        <a class="btn">learn how you can tell your instructor about openstax!</a>
+    </div>
+</section>
+<div class="block">
+    <section class="details">
+        <h1>Details</h1>
+        {{{bookInfo.allAuthorsHtml}}}
+    </section>
+    <section class="errata">
+        <h1>Errata</h1>
+        <p>
+            All OpenStax textbooks undergo a rigorous review process. However, like
+            any professional-grade textbook, errors sometimes occur. The good part
+            is, since our books are web-based, we can make updates periodically. If
+            you have a correction to suggest, submit it here. We'll review your
+            suggestion and make necessary changes.
+        </p>
+        <div class="links">
+            <a>Errata list &gt;</a>
+            <a>Suggest a correction &gt;</a>
```

Comment:
If the `>` symbols are for stylistic purposes only, it may be more appropriate to add with a CSS pseudo element.  Otherwise it's quite awkward for users of screen readers.